### PR TITLE
KEYCLOAK-17407 - Add Retry

### PR DESCRIPTION
**Pull request by Aboullos**
Test works fine in local, but in Jenkins it needs a wait of 1 second

### CODE REVIEW

**Comment by hmlnarik** [comment]
Can `Thread.sleep` be avoided and rather done via e.g. Retry - 10 retries in 1 second would do the job without unnecessary waiting.
Also please file a JIRA.

**Comment by Aboullos** [comment]
The test passed with the changes in this build:
new-jenkins.com/job/universal-test-pipeline-server/152

I applied the retry to the old code too

File: testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/authorization/DisableAuthorizationSettingsTest.java [hunk 1]
```diff
@@ -16,11 +16,11 @@
  */
 package org.keycloak.testsuite.console.authorization;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```
Comment (hmlnarik) [inline testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/authorization/DisableAuthorizationSettingsTest.java:19]: Please avoid star imports.

**Review by hmlnarik — CHANGES_REQUESTED** [review]
Thank you for the PR. Looks good apart from a minor thing.

**Review by mhajas — APPROVED** [review]
@Aboullos Looks good to me. Thanks

**Review by pdrozd — APPROVED** [review]

**Pull request merged**